app/views/tasks/_todo.html.erb

```diff
@@ -1,7 +1,7 @@
 <div class='w-[100%] border border-black rounded-lg p-4 m-1 flex justify-between items-center'>
   <div class='flex items-center'>
-    <%= task.status ? '<input type="checkbox" disabled checked/>'.html_safe : '<input type="checkbox" disabled/>'.html_safe %>
-    <%= task.status ? "<p class='line-through text-gray-400 ml-2'>#{task.title}</p>".html_safe : "<p class='ml-2'>#{task.title}</p>".html_safe %>
+    <%= task.done? ? '<input type="checkbox" disabled checked/>'.html_safe : '<input type="checkbox" disabled/>'.html_safe %>
```

Comment:
You might find view helper can help you on this. Create a helper to define logics and include in view to reduce conditions. 

See https://medium.com/@acushla4real/rails-view-helpers-best-practices-guide-d7d7b6db3db4